recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "requests" %}
-{% set version = "2.28.0" %}
-{% set hash = "d568723a7ebd25875d8d1eaf5dfa068cd2fc8194b2e483d7b1f7c81918dbec6b" %}
+{% set version = "2.28.1" %}
+{% set hash = "7c5599b102feddaa661c826c56ab4fee28bfd17f5abca1ebbe3e7f19d7c97983" %}
 {% set build_number = "0" %}
 
 package:
```

Comment:
There is chardet-normilizer to be updated https://github.com/psf/requests/blame/v2.28.1/setup.py#L62

Reply:
`python` from `test/requires` can be removed. It's already in `run`

Reply:
yes, please update to chardet-normalizer to bounds >=2,<3 instead

Reply:
yes, python dependency shouldn't be necessary in test/require anymore.

Reply:
as you rebuilding it anyway, please do thost cleanups too.
Thanks